src/apiCalls.js

```diff
@@ -0,0 +1,18 @@
+import endpoints from './endpoints.js';
+
+export const postNewEmailSubscriber = (newEmailSubscriber) => {
+  const { first_name, last_name, state_name, email, language } = newEmailSubscriber;
+
+  return fetch(endpoints.submit, {
```

Comment:
What do you think about just using the actual endpoint `http://localhost:3001/api/v1/users`?

Reply:
I am very attached to my endpoints file!  I like that if I need to do anything related to an endpoint I have just one place I need to go instead of finding the component that is making the fetch call.  I also find it really helpful to have that one document that serves as reference when tracking requests and responses & it gets used for both development & especially testing.

Reply:
What do you think about calling it `endpoints.users`? It would be easier to understand which endpoint it's hitting since that information would be living outside of this function.

Reply:
Yes, definitely!  Good idea!